projects/models.py

```diff
@@ -1,3 +1,174 @@
+from datetime import timedelta
+from uuid import uuid4
+
 from django.db import models
+from django.core.validators import MinValueValidator, MaxValueValidator
+from django.core.exceptions import ValidationError
+
+from simple_history.models import HistoricalRecords
+
+from startups.models import StartUpProfile
+from investors.models import InvestorProfile
 
 # Create your models here.
+class Project(models.Model):
+    """Project model
+    
+    Fields:
+    - project_id(UUIDField)
+    - startup(ForeignKey): id of startup owning the project
+    - investor(ManyToManyField): investors sponsoring the project
+    - title(CharField): project name
+    - risk(FloatField): project risk value (0.0 - 1.0)
+    - description(TextField): project description
+    - business_plan(URLField): URL link to business plan file
+    - amount(DecimalField): finances needed
+    - status(IntegerField): project status choice field(seeking, in_progress, final_call, closed)
+    - duration(DurationField): project duration
+    - created_at(DateTimeField)
+    - updated_at(DateTimeField
+    - history: HistoricalRecords object for tracking updates"""
+    class ProjectStatus(models.IntegerChoices):
+        """IntegerChoices ProjectStatus class
+        
+        - SEEKING = 1: active, no investors yet
+        - IN_PROGRESS = 2:  active, some investors involved
+        - FINAL_CALL = 3: active, almost funded
+        - CLOSED = 4: inactive"""
+
+        SEEKING = 1
+        IN_PROGRESS = 2
+        FINAL_CALL = 3
+        CLOSED = 4
+
+    project_id = models.UUIDField(
+        primary_key=True, default=uuid4, editable=False)
+    startup = models.ForeignKey(
+        StartUpProfile, related_name='projects', on_delete=models.CASCADE)
+    investors = models.ManyToManyField(
+        InvestorProfile, related_name='projects', blank=True, through="Subscription")
+    title = models.CharField(max_length=255)
```

Comment:
To ensure the uniqueness of the project. What do you say?   =>

class Project(models.Model):
    ...
    title = models.CharField(max_length=255, unique=True)